third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -333,12 +351,27 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 				ForceNew:  true,
 				Sensitive: true,
 			},
+			"source_ip_address": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: externalReplicaConflictKeys,
+				ValidateFunc:  validation.SingleIP(),
+			},
+			"source_port": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: externalReplicaConflictKeys,
+				InputDefault:  "3306",
```

Comment:
We don't use `InputDefault` anywhere else in the provider- is there a reason this shouldn't just be `Default`?

Reply:
`Default` property will always set that parameter, but we don`t need it, because of conflicts with `settings`